crates/forge-runner/src/sierra_test_code_path.rs

```diff
@@ -0,0 +1,35 @@
+use crate::compiled_runnable::CompiledTestCrateRunnable;
+use anyhow::Context;
+use anyhow::Result;
+use cairo_lang_sierra::program::VersionedProgram;
+use camino::Utf8PathBuf;
+use fs4::FileExt;
+use std::fs::File;
+use std::io::BufWriter;
+
+pub struct SierraTestCodePath {
+    pub path: Utf8PathBuf,
```

Comment:
This shouldn't be `pub` bcs this way we cannot guarantee that this is a valid path. 
We should have a function which copies sierra and retruns the path (not path constructor imo, a path is a path), this way we would guarantee on type level that every `SierraCodePath` actually points to a sierra file.

Reply:
Done